src/exberry_int/integration_exberry.py

```diff
@@ -73,7 +83,8 @@ async def request_session(api_key: str, secret_str: str):
                 'signature': signature
             }
         }
-        await enqueue_outbound(create_session)
+        await ws.send_json(create_session)
+        LOG.info("Waiting for session request to be confirmed...")
```

Comment:
Where is this wait occurring?  The `send_json` appears to be the last unique thing in this control flow.

Reply:
Also.... do you know if this delays the startup of the integration's web server? If it does, by how much?

Reply:
The next thing that should happen after the `send_json` is the response to the "create session" request, which is a "session created" message from Exberry (https://github.com/digital-asset/daml-dit-integration-exberry/pull/13/files#diff-5ce9e982e5ab6264daf5b96b7a4a3e784db120f5ce8aca14bbc25a0c29847d9eR234). When that message is received, the `session_started` event that is blocking all outbound messages from being sent is set to active, starting the flow of outbound messages.

Reply:
I am not sure what you mean by the integration's web server - the incoming websocket messages from Exberry still start immediately and still create contracts as they come in, it just delays the outbound messages by a short amount of time - and incoming contract creates are still added to the outbound queue.

Reply:
Ok... I see the flow now.

So you're aware, every integration has a web server that's used both for inbound webhook requests, but also for k8s readiness and liveness checks. That web server isn't guaranteed to start until the integration entry point has returned, so if that process takes a while, it can cause issues getting the integration started. (But I don't think that's going to happen here.)